components/match2/wikis/trackmania/match_legacy.lua

```diff
@@ -0,0 +1,158 @@
+---
```

Comment:
you use double quotationmarks here
pls change it to single ones
see #2250 for reference

Reply:
https://github.com/Liquipedia/Lua-Modules/pull/2259/commits/f9e111ad9d27e71a7a2d361e583f841a424abec5